Cargo.toml

```diff
@@ -1,14 +1,22 @@
 [package]
 name = "ic-verify-bls-signature"
-version = "0.2.0"
+version = "0.3.0"
 edition = "2021"
 license = "Apache-2.0"
 description = "A library for handling BLS signatures"
 
 [dependencies]
-bls12_381 = { version = "0.7", default-features = false, features = ["groups", "pairings", "alloc", "experimental"] }
+bls12_381 = { version = "0.7", default-features = false, features = ["groups", "pairings", "experimental"] }
 pairing = "0.22"
-lazy_static = "1"
-rand = "0.6"
-sha2 = "0.9"
+sha2 = { version = "0.9", default-features = false }
+rand = { version = "0.6", default-features = false }
+
+lazy_static = { version = "1", optional = true, default-features = false }
```

Comment:
nit: There is no `default` feature in `lazy_static`, so not sure the `default-features = false` makes a difference has (currently) any effect.

Reply:
I believe you are correct. I'll remove this